homeassistant/components/samsungtv/__init__.py

```diff
@@ -1 +1,61 @@
 """The Samsung TV integration."""
+import voluptuous as vol
+from wakeonlan import BROADCAST_IP
+
+from homeassistant.const import (
+    CONF_BROADCAST_ADDRESS,
+    CONF_HOST,
+    CONF_MAC,
+    CONF_NAME,
+    CONF_PORT,
+    CONF_TIMEOUT,
+)
+import homeassistant.helpers.config_validation as cv
+
+from .const import DEFAULT_NAME, DEFAULT_TIMEOUT, DOMAIN
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.All(
+            [
+                vol.Schema(
+                    {
+                        vol.Required(CONF_HOST): cv.string,
+                        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+                        vol.Optional(CONF_PORT): cv.port,
+                        vol.Optional(CONF_MAC): cv.string,
+                        vol.Optional(
+                            CONF_TIMEOUT, default=DEFAULT_TIMEOUT
```

Comment:
Do we really need an option for the timeout? Can't we set a sane default?

Reply:
I don't need, its there since the early days.